src/competition/Human.java

```diff
@@ -0,0 +1,19 @@
+package competition;
+
+ public class Human implements Participant {
+    private String name;
+
+    public Human(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public void run() {
+        System.out.println(name + " runs");
```

Comment:
the implementations are the same in all classes. The code duplicates everywhere. Change the implementation for the participants or move the implementation to the default method